keanu-python/keanu/base.py

```diff
@@ -18,6 +18,9 @@ def __getattr__(self, k):
 
             raise AttributeError("{} has no attribute {}".format(self.__class__, k))
 
+        if python_name == "_get_object_id":
```

Comment:
This is a common error case that I encountered as I was devving, so I thought it would be valuable to catch it and give a more informative error to the user.

There are other similar ones that I haven't caught yet, which arise when you put a python type where a java type is expected or vice versa. Your work on type annotations will help a lot with this too.